plugins/woocommerce/tests/e2e-pw/envs/default-pressable/playwright.config.js

```diff
@@ -3,12 +3,11 @@ const { devices } = require( '@playwright/test' );
 
 config = {
 	...config,
-	retries: 0,
 	projects: [
 		{
 			name: 'default',
 			use: { ...devices[ 'Desktop Chrome' ] },
-			testMatch: '**basic.spec.js',
+			testMatch: '**.spec.js',
```

Comment:
Is there some better way to run all e2e tests here?

Reply:
Removing `testMatch` entirely will run all the tests.